contracts/Liquidator/Liquidator.sol

```diff
@@ -402,15 +401,25 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, Liqu
     /// @dev Distribute seized collateral between liquidator and protocol share reserve
     function _distributeLiquidationIncentive(
         IVToken vTokenCollateral,
-        uint256 siezedAmount
+        uint256 seizedAmount
     ) internal returns (uint256 ours, uint256 theirs) {
-        (ours, theirs) = _splitLiquidationIncentive(siezedAmount);
+        (ours, theirs) = _splitLiquidationIncentive(seizedAmount);
         if (!vTokenCollateral.transfer(msg.sender, theirs)) {
             revert VTokenTransferFailed(address(this), msg.sender, theirs);
         }
 
-        if (!_redeemUnderlying(address(vTokenCollateral), ours)) {
-            pendingRedeem.push(address(vTokenCollateral));
+        if (ours > 0 && !_redeemUnderlying(address(vTokenCollateral), ours)) {
+            bool found;
```

Comment:
I think we can avoid the extra variable `found`, defining the `index` variable out and before the `for` loop, and using it in the `if` statement this way:
```
if (index == pendingRedeem.length) {
    pendingRedeem.push(address(vTokenCollateral));
}
```

Reply:
Fixed [here](https://github.com/VenusProtocol/venus-protocol/pull/298/commits/a2e94c94e0aedf478a8bcee1efdecb1958bbe20c)